### DISCUSSION TITLE
please help tell me if this is a scammer

### DISCUSSION THREAD

**Discussion by ckoretsky** [dogecoin/dogecoin#3354] [Help needed]
i would like if someone could tell me if this is a scam.  i will post screenshots of the conversation.  thanks
![IMG-0108](https://github.com/dogecoin/dogecoin/assets/21155303/bbe06b20-5d05-49cc-9934-5e0f44d5dc0f)
![IMG-0107](https://github.com/dogecoin/dogecoin/assets/21155303/9a7fccd3-6886-479d-a12e-25ead126dd7f)
![IMG-0102](https://github.com/dogecoin/dogecoin/assets/21155303/bc2d0697-f20e-4ca0-981d-c55da2ac7ff9)
![IMG-0103](https://github.com/dogecoin/dogecoin/assets/21155303/5672fd6c-c982-47be-8f1f-acce76f7bb1f)
![IMG-0105](https://github.com/dogecoin/dogecoin/assets/21155303/91898c6e-b410-40f1-bdc5-6aa03c4d8e26)
![IMG-0106](https://github.com/dogecoin/dogecoin/assets/21155303/599cc9d1-1d2f-4aab-a167-a5f9c5b53927)
![IMG-0107](https://github.com/dogecoin/dogecoin/assets/21155303/41e9a9b5-88a2-4667-b0d2-e9da06237f22)
![IMG-0108](https://github.com/dogecoin/dogecoin/assets/21155303/6dd552fc-2a7e-4721-b6ea-a666fef76e36)

**Comment by patricklodder** [comment]
It doesn't make much sense but I'm not sure what your discussion is about. What is the context?

Blockchain "walls" is not really a thing though, at least assuming you're talking about Bitcoin since the screenshot of blocks mined is from Bitcoin: the marked block is: https://mempool.space/block/00000000000000000008ea2ea94778dfd157bc5b34fc4572cf8144ee9c15491f

**Reply by ckoretsky** [reply]
I lost 1500 of bitcoin. He was helping me get it back.

**Reply by ckoretsky** [reply]
Is he right about me being able to get my money back?

**Reply by patricklodder** [reply]
> Is he right about me being able to get my money back?

Considering that you said that "Coinbase Support on Twitter" keeps on taking your coins, I think not, but I can be wrong. I think it is likely that your coin is gone, because there are a lot of scams on Twitter/X claiming to be Coinbase Support (but they aren't.)

If coin was withdrawn from Coinbase but never arrived, then the only party that can help you is the Coinbase Support from their website. Contact them through their formal support channel here: https://help.coinbase.com/en/contact-us?product=coinbase

If the coin was withdrawn and received, and later lost from your wallet, you'll need to check with your wallet support. Also not on social media, but through the wallet provider's formal channels if they have any.
